unfoldingWord/translationCore
# Feature bspidel 3222

#### This pull request addresses:

Epic 3222 with tasks which truncate books and language on Project Cards

#### How to test this pull request:

Unit test for Project Card exercises code when no truncation is required.
Special test case handles actual truncation.

To see results,
  open tC and import a project with a long language as included in original story: rmr-x-bsa_mat_text_ulb


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3373)
